biofefi/feature_importance/interpreter.py

```diff
@@ -223,6 +228,8 @@ def _ensemble_feature_importance(self, feature_importance_results):
                         )
                         ensemble_results[ensemble_type] = majority_vote_results
 
-        self._logger.info(f"Ensemble feature importance results: \n{ensemble_results}")
+            self._logger.info(
+                f"Ensemble feature importance results: \n{ensemble_results}"
```

Comment:
```suggestion
                f"Ensemble feature importance results: {os.linesep}{ensemble_results}"
```
This is how you get the line separator for whatever OS the code is running on. Windows' one is different to mac and linux